src/Cli/dotnet/Commands/CliCommandStrings.resx

```diff
@@ -1520,7 +1520,7 @@ Tool '{1}' (version '{2}') was successfully installed. Entry is added to the man
     <comment>{0} is the file path and line number.</comment>
   </data>
   <data name="InvalidDirectiveName" xml:space="preserve">
-    <value>The directive at '{2}' should contain a name without special characters and an optional version separated by '{1}' like '#:{0} Abc{1}Xyz'.</value>
+    <value>The directive at '{2}' should contain a name without special characters and an optional version separated by '{1}' like '#:{0} Name{1}Version'.</value>
```

Comment:
This error can be reported for `#:property` as well though, where the error text `#:property Name=Version` might be confusing?

Reply:
I will go back and adjust the `<data name="..."` value to indicate the scenarios this message is used for.

Reply:
Actually, I see that `DotnetProjectConvertTests.Directives_InvalidName` is expecting some property directives to report this diagnostic? In which case, the suggestion to use an optional version probably doesn't make sense, this message could probably suggest using "an optional value" and "Name=Value" instead, to generalize.

Reply:
Right, the message shouldn't mention version in the text, I've missed that. It's true properties have their own diagnostic messages but also can use this one as well. Using "optional value" and "Name{1}Value" sounds good to me. Thanks.